src/main/java/com/gal/trivia/entity/Question.java

```diff
@@ -1,30 +1,29 @@
 package com.gal.trivia.entity;
 
-import lombok.Data;
+import lombok.*;
 
 import javax.persistence.*;
 import java.util.Date;
-import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
 
 @Entity
-@Data
+@Getter
```

Comment:
In place of Getter, Setter AllArgsConstructor we can use @Data which supports all 3 together.

Reply:
It make sense.. Thanks will fix it